### DISCUSSION TITLE
能支持导入/导出svg吗

### DISCUSSION THREAD

**Discussion by nickolboy** [JessYan0913/pictode#23] [Ideas]
### Feature Request / 功能请求

**问题描述 / Problem Description**
需要导出svg，插入到富文本编辑器中展现。并且能从编辑器打开svg再次编辑

**Comment by JessYan0913** [comment]
支持导入svg，但是Konva本身编辑svg可能会有些问题

**Reply by nickolboy** [reply]
还是希望能够支持，想做成将图片嵌入文档的功能。保存的时候，输出成svg格式内容，在富文本中可以显示。编辑文档中的图片时，将这个svg用pictode打开，能够再次编辑。现在toDataURL()只能输出png格式，无法再次编辑
